README.md

```diff
@@ -73,6 +73,7 @@ Please file an issue: https://github.com/sourcegraph/sourcegraph-jetbrains/issue
 
 ## Version History
 
+- v1.1.2 - Updated the search action to trigger the open action when no text is selected.
```

Comment:
As a user I find this behavior confusing. If I activate the search shortcut and there is no selection, I expect it to do nothing. Opening the file is confusing because I am expecting search results

Reply:
I see. How about if it opened to the repository with the search input active instead of opening to the file position with the current cursor selected?